Fault.py

```diff
@@ -17,11 +17,21 @@ def __init__(self, tlc, severity, suffix, okValue, faultValue, description,
         self.name = name
         self.okValue = float(okValue) if okValue else None
         self.faultValue = float(faultValue) if faultValue else None
+        self.suffix = suffix
 
         self.pv: PV = PV(prefix + ":" + suffix, connection_timeout=PV_TIMEOUT)
 
     def isFaulted(self):
-        if self.pv.status is None:
+
+        """
+        Dug through the pyepics source code to find the severity values:
+        class AlarmSeverity(DefaultIntEnum):
+            NO_ALARM = 0
+            MINOR = 1
+            MAJOR = 2
+            INVALID = 3
```

Comment:
So I thought this only applied to our CUDSEVR pvs. Is this the general alarm severity for ALL pvs?

Reply:
It's like, the native epics severity that has nothing to do with the values in our CSV haha But invalid here means that while the PV is reading out values, we can't trust those values because they're invalid in some way

Reply:
So just to confirm, does this have anything to do with pvName.SEVR?

Reply:
Oh I dunno. Honestly I have no clue haha